PdCore/src/org/puredata/android/io/AudioParameters.java

```diff
@@ -41,14 +41,14 @@
 	 */
 	public static synchronized void init(Context context) {
 		if (impl != null) return;
-		if (Properties.version > 16 && context != null) {
+		if (Properties.version > Build.VERSION_CODES.JELLY_BEAN && context != null) {
 			impl = JellyBeanMR1OpenSLParameters.getParameters(context);
-		} else if (Properties.version > 16) {
-			Log.w("AudioParameters", "Initializing audio parameters with null context on Android 4.2 or later.");
+		} else if (Properties.version > Build.VERSION_CODES.JELLY_BEAN) {
+			Log.w(TAG, "Initializing audio parameters with null context on Android 4.2 or later.");
 			impl = new BasicOpenSLParameters(64, 64);
-		} else if (Properties.version == 16) {
+		} else if (Properties.version == Build.VERSION_CODES.JELLY_BEAN) {
 			impl = JellyBeanOpenSLParameters.getParameters();
-		} else if (Properties.version > 8) {
+		} else if (Properties.version > Build.VERSION_CODES.FROYO) {
```

Comment:
No good reason.
I've changed it to API level 10 now and also did the other changes you requested.